.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+ [ ] I have searched the [PR](https://github.com/Marketing-Automation-Systems/jellyfish/pulls/) of this repository and believe that this is not a duplicate.
+
+<!-- Checked checkbox should look like this: [x] -->
+
+**Describe the PR**
+
+This PR closes #ISSUE_NUMBER
+
+<!-- Provide a description of the changes proposed in the pull request -->
+
+<!--
+  Thank you very much for contributing to the Jellyfish team by creating an issue! ❤️
+  To avoid duplicate PR we ask you to check off the following list.
+-->
```

Comment:
Maybe move this to the top of the file? I feel like this part is out of context here.

Reply:
Changed.